FreeRTOS/Test/CMock/message_buffer/message_buffer_utest.c

```diff
@@ -0,0 +1,670 @@
+/*
```

Comment:
Please add @ coverage tags to this file so that the lcov / lcovhtml targets will work correctly.

Reply:
This is fixed in coverage filter script. So not required.